src/types/express.d.ts

```diff
@@ -235,6 +235,11 @@ declare module "express-session" {
     formErrors: { [key: string]: { text: string } };
     decision: { status: string; notes: string };
     formValuesValidationError: RequestBody;
+    resourceDetails?: {
```

Comment:
this crept in after merging orgin/main - not sure why its showing a diff? Anyone?